plaso/parsers/sqlite_plugins/windows_eventtranscript.py

```diff
@@ -0,0 +1,224 @@
+# -*- coding: utf-8 -*-
+"""SQLite parser plugin for Windows Diagnosis EventTranscript database file."""
+import json
+
+from dfdatetime import filetime as dfdatetime_filetime
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import definitions
+from plaso.parsers import sqlite
+from plaso.parsers.sqlite_plugins import interface
+
+
+class EventTranscriptEventData(events.EventData):
+  """Windows diagnosis EventTranscript event data.
+
+  Attributes:
+    sid (str): Windows Security ID of a user account.
+    timestamp (int): Event timestamp in UTC
+    raw_payload (str): JSON payload specific to event name
```

Comment:
I opt we keep raw_payload, ext and data out of the the event data object.
this can be expensive to store if the data is large